File: easy-solution/galaxy4276/9.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0: return False
        palin_str = str(x)
        reversed_palin_str = list(reversed(list(palin_str)))

        for i in range(0, len(palin_str)):
          if palin_str[i] != reversed_palin_str[i]:
            return False
        return True

File: easy-solution/galaxy4276/9.rs
impl Solution {
    pub fn is_palindrome(x: i32) -> bool {
        if x < 0 {
            return false;
        }
        let chars: Vec<char> = x.to_string().chars().collect();
        let reversed_chars: Vec<char> = x.to_string().chars().rev().collect();
        for idx in 0..chars.len() {
            if chars[idx] != reversed_chars[idx] {
                return false;
            }
        }
        true
    }
}
